src/frameworks/next/index.ts

```diff
@@ -208,13 +209,21 @@ export async function build(dir: string): Promise<BuildResult> {
     headers.push(...headersFromMetaFiles);
 
     if (appPathsManifest) {
-      const unrenderedServerComponents = getNonStaticServerComponents(
+      let unrenderedServerComponents = getNonStaticServerComponents(
         appPathsManifest,
         appPathRoutesManifest,
         prerenderedRoutes,
         dynamicRoutes
       );
 
+      if (unrenderedServerComponents.includes("/_not-found")) {
```

Comment:
nit, if you store index then the delete will be easier `delete unrenderedServerComponents[index]` then it can remain a const

Reply:
Even better thanks